gravitee-apim-console-webui/src/management/settings/groups/group/add-members-dialog/add-members-dialog.component.html

```diff
@@ -0,0 +1,73 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<h2 mat-dialog-title>Add members</h2>
+<mat-dialog-content>
+  <form [formGroup]="addMemberForm" class="add-member__form">
+    <mat-form-field aria-hidden="false" aria-label="Select default API role">
+      <mat-label>Default API Role</mat-label>
+      <mat-select [disabled]="!canChangeDefaultAPIRole()" formControlName="defaultAPIRole">
```

Comment:
disabled state should be handled in the form itself I think. I think you should have a warning in the browser console.
It should be done using something like this in the component ts.

```typescript
     defaultAPIRole : new FormControl({ value: myValue, disabled: true/false }),
```

Disable state changes can even be computed in the ts file according to form events or component events I think.